### ISSUE TITLE
Retrain Automod

### ISSUE THREAD

**Issue by CalderJohnson** [uwindsorcss/ai-automod#3]
#### Describe Problem
The automod needs to be retrained to improve accuracy and usefulness. 

#### Suggest Solution
Modify the training loop and adjust the training dataset then conduct another training run to enhance accuracy.

#### Additional Details
Some data augmentation and sifting may be required for this. I've added Damian, Jovic and Kevin to this issue as well for potential assistance in this department.
